packages/debugger/package.json

```diff
@@ -49,11 +49,10 @@
     "@truffle/migrate": "^3.2.55",
     "@truffle/resolver": "^7.0.39",
     "@truffle/workflow-compile": "^3.2.41",
-    "babel-core": "^6.26.0",
-    "babel-loader": "^7.1.2",
-    "babel-plugin-transform-object-rest-spread": "^6.26.0",
-    "babel-plugin-transform-runtime": "^6.23.0",
-    "babel-preset-env": "^1.6.1",
+    "@babel/core": "^7.16.0",
```

Comment:
Whoa, you managed to upgrade to Babel 7 and get things working, amazing!

Of course, that leaves the question of if we even still *need* babel; maybe that can be torn out?  Or maybe that'd be better saved for a later PR.

Reply:
I think a later PR, there are some things its doing to the code that would need a bit more of cleanup I think.